doc/api/n-api.md

```diff
@@ -4678,7 +4678,7 @@ napi_status napi_get_cb_info(napi_env env,
 * `[in-out] argc`: Specifies the length of the provided `argv` array and
   receives the actual count of arguments. `argc` can
   optionally be ignored by passing `NULL`.
-* `[out] argv`: Buffer to which the `napi_value` representing the arguments are
+* `[out] argv`: Array of `napi_values` to which the arguments will be
```

Comment:
```suggestion
* `[out] argv`: C array of `napi_values` to which the arguments will be
```

(because this is a situation in which “array” manages to be both more precise and more ambiguous – it’s important that this specifically does *not* refer to handle for a JS array)